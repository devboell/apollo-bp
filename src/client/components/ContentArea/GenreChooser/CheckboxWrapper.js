import styled from 'styled-components'

const CheckboxWrapper = styled.div`

  margin-right: 10px;
  > label {
    font-weight: bold;
    margin-left: 5px;
    vertical-align: -1px;
  }
`

export default CheckboxWrapper
