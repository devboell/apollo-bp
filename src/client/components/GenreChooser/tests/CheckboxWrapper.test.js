import React from 'react'
import 'jest-styled-components'

import CheckboxWrapper from '../CheckboxWrapper'

it('CheckboxWrapper, styled component snapshot', () => {
  const wrapper = shallow(<CheckboxWrapper />)
  expect(wrapper).toMatchStyledComponentsSnapshot()
})
