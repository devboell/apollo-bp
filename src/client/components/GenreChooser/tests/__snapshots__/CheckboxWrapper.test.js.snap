// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckboxWrapper, styled component snapshot 1`] = `
.bePfnC {
  margin-right: 10px;
}

.bePfnC > label {
  font-weight: bold;
  margin-left: 5px;
  vertical-align: -1px;
}

<div
  className="sc-bdVaJa bePfnC"
/>
`;
